string name
sensor_msgs/Image mask
float32 score
int64[2] pos
int64 area
string[] additional_names
int64[4] bbox # [x1, y1, x2, y2]
int64 bbox_area
float64 angle
int64 width
int64 length
int64[8] rotated_rect # [x1, y1, x2, y2, x3, y3, x4, y4]
int64 rotated_rect_area
